stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -368,10 +368,22 @@ public class ChannelListView : FrameLayout {
         this.loadingView.isVisible = true
     }
 
+    @Deprecated(
+        message = "Deprecated in favor of new logic for setChannels. To set the loading item call set channels with" +
+            "the loading item already in the list. Show loading more function is race condition prone, which can" +
+            "cause the list to apply an old state of channels.",
+        level = DeprecationLevel.WARNING
```

Comment:
Since it's deprecated with a warning but we still have the old behavior, can we, within the body, just do what the message says?

Just add the list of items with the loading item?